lib/results.js

```diff
@@ -97,7 +99,10 @@ Results.prototype._watch = function (t) {
     var self = this;
     var write = function (s) { self._stream.queue(s) };
     t.once('prerun', function () {
-        write('# ' + t.name + '\n');
+        var premsg = '';
+        if (t._todo) premsg = 'TODO ';
+        if (t._skip) premsg = 'SKIP ';
```

Comment:
can a test be both todo and skip?

Reply:
Here `skip` takes precidence https://github.com/substack/tape/blob/db1aa0682204265f9765c7ddea931726e01da8b8/lib/test.js#L87-L89 so I thought we might not need an else. When the user passes in both `todo` and `skip` in `opts`, skip takes precidence.

Reply:
It seems useful to keep the else here, just in case someone calls into this directly.